var/spack/repos/builtin/packages/cmor/package.py

```diff
@@ -13,8 +13,9 @@ class Cmor(AutotoolsPackage):
     standard model experiments."""
 
     homepage = "https://cmor.llnl.gov"
-    url = "https://github.com/PCMDI/cmor/archive/3.4.0.tar.gz"
+    url = "https://github.com/PCMDI/cmor/archive/3.6.1.tar.gz"
 
+    version('3.6.1', sha256='991035a41424f72ea6f0f85653fc13730eb035e63c7dff6ca740aa7a70976fb4')
```

Comment:
Confirmed the sha256

Reply:
Thanks, added Adam for review since he did most updates of the cmor!